app/demo-stm32h7-nucleo/app-h743.toml

```diff
@@ -37,7 +37,6 @@ name = "drv-stm32xx-i2c-server"
 stacksize = 1048
 features = ["h743"]
 priority = 2
-max-sizes = {flash = 16384, ram = 4096}
```

Comment:
We hit the flash size limit for the I2C server task, and @cbiffle said it would be fine to just remove the limits so that we don't have to keep playing guess-a-number forever.